.github/workflows/ci_test.yml

```diff
@@ -1,32 +1,31 @@
 name: CI
 
-on:
-  push:
-    branches: [ master, develop ]
-  pull_request:
-    branches: [ master, develop ]
+on: [push]
 
 jobs:
-  build:
-    runs-on: ubuntu-latest
-    env:
-      frontend-folder: .
-
+  cypress:
+    name: Cypress run
+    runs-on: self-hosted
+    strategy:
+        fail-fast: false
+        matrix:
+            containers: [1, 2, 3, 4, 5]
     steps:
-      - uses: actions/checkout@v2
-
-      - name: Instal npm packages
-        run: npm install
-        working-directory: ${{env.frontend-folder}}
+        - name: Checkout
+          uses: actions/checkout@v2
 
-      - name: Run application
-        run: npm start &
-        working-directory: ${{env.frontend-folder}}
+        - name: Install npm packages
+          run: npm install
```

Comment:
Może dodalibyśmy też cachowanie zależności?
Bo z tego co patrzyłem, to github actions, mają wbudowaną taką opcję

https://docs.github.com/en/actions/guides/caching-dependencies-to-speed-up-workflows

Wtedy myślę, że będzie jeszcze szybciej

Reply:
Jak masz self-hosted runner to zawsze się cachuja w sensie  się nie usuwają, zobacz ze teraz npm install trwa 20 sekund, on zawsze musi przejść przez te paczki i sprawdzić czy jest ok.